src/Sluggable/Handler/RelativeSlugHandler.php

```diff
@@ -120,6 +111,10 @@ public function transliterate($text, $separator, $object)
             $this->originalTransliterator,
             [$text, $separator, $object]
         );
+        $result = call_user_func_array(
+            $this->sluggable->getUrlizer(),
+            [$result, $separator, $object]
+        );
```

Comment:
Calling the urlizer here is the only way I could work out fixing this test failure:

```sh
1) Gedmo\Tests\Sluggable\Handlers\BothSlugHandlerTest::testSlugUpdates
   Failed asserting that two strings are identical.
   --- Expected
   +++ Actual
   @@ @@
   -'web/developer/upd-gedi'
   +'web/developer/upd gedi'

/tests/Gedmo/Sluggable/Handlers/BothSlugHandlerTest.php:64
```